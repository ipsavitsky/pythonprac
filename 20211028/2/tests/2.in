print(*list(slide(sorted([7, 14, 12312, 43, 234, 1, 3421]), 2)))