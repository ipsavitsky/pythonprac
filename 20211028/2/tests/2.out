1 7 7 14 14 43 43 234 234 3421 3421 12312 12312
